### PR TITLE
Fix tfrs listwise training issue

### PR DESCRIPTION
**Pull request by cyita**
Related issue #4505

### CODE REVIEW

**Comment by cyita** [comment]
Jenkins passed http://10.112.231.51:18888/job/BigDL-Orca-PR-Validation/646/
http://10.112.231.51:18888/job/BigDL-Orca-PR-Validation/647/
http://10.112.231.51:18888/view/BigDL-2.0-PR/job/BigDL-Friesian-PR-Validation/65/

File: python/orca/example/learn/tf2/dcn/dcn_parquet.py [hunk 1]
```diff
@@ -174,17 +174,16 @@ def compute_loss(self, features, training=False):
 
     def model_creator(config):
         model = DCN(use_cross_layer=True, deep_layer_sizes=[192, 192])
-        from bigdl.friesian.learn.models import tfrs_model
-        model = tfrs_model(model)
+        from bigdl.friesian.learn.friesianTFRSModel import FriesianTFRSModel
```
Comment (jason-dai) [inline python/orca/example/learn/tf2/dcn/dcn_parquet.py:177]: change to `from bigdl.friesian.models import TFRSModel`

**Comment by jason-dai** [comment]
LGTM

**Comment by cyita** [comment]
Jenkins passed http://10.112.231.51:18888/view/BigDL-2.0-PR/job/BigDL-Friesian-PR-Validation/67/

**Pull request merged**